netjson-monitoring.lua

```diff
@@ -458,6 +458,20 @@ for radio_name, radio in pairs(wireless_status) do
     end
 end
 
+
+function is_part_of_bridge(name)
+    for _, interface in pairs(network_status) do
+      if interface['bridge-members'] then
+        for _, key in pairs(interface['bridge-members']) do
+          if key  == name then
+            return true
+          end
+        end
+      end
+    end
+    return false
+end
```

Comment:
indentation is inconsistent, the first block is indented with 4 spaces but should be 2